fli-iam/shanoir-ng
# [Fli iam#2059] Use new backend execution & pipeline APIs in Shanoir front

# Changes
- All VIP API call are now made through MS datasets backend
  - `.../datasets/vip/pipeline`
  - `../datasets/vip/execution` 
- Use Spring 5 Web Client (see VipClientService.java)
- Write VIP handler logs into shanoir-vip-handler.log

# How to test
- Deploy on OFSEP qualif
- Check that datasets Swagger doc is up to date
- Execute OFSEp Seq Id & ct-tiqua VIP pipelines on multiple datasets through 
   - Shanoir website
   - shanoir-downloader Python script
- Check that execution status is correctly updated
- Check that execution logs (stdout, stderr) can be fetched in execution monitoring view 
- Check that result are well processed
- Check that handler logs are written in /var/log/dk/ofsep/qualif/shanoir-ng/shanoir-vip-handler.log
